### ISSUE TITLE
Installation docs are incorrect

### ISSUE THREAD

**Issue by darylknight** [craftcms/commerce-docs#9]
I currently don't have access to the plugin store ( `cURL error 7: Failed to connect to api.craftcms.com port 443: Timed out` ), so I'm trying to follow these docs to install Commerce via composer. Following the line that says to run `composer require craftcms/commerce` gives the following error:

```
[InvalidArgumentException]
Could not find a version of package craftcms/commerce matching your minimum-stability (stable).
Require it with an explicit version constraint allowing its desired stability.
```

**Comment by lukeholder** [comment]
Since commerce 2 is in beta you will need to remove your minimum stability of stable in composer or choose a different minimum.

See https://getcomposer.org/doc/04-schema.md#minimum-stability

**Comment by lukeholder** [comment]
Fixed in 2c7792f2bd2a231e443c9f5c4c4c044189f1a1f6
